ext/strscan/strscan.c

```diff
@@ -2290,4 +2330,6 @@ Init_strscan(void)
     rb_define_method(StringScanner, "fixed_anchor?", strscan_fixed_anchor_p, 0);
 
     rb_define_method(StringScanner, "named_captures", strscan_named_captures, 0);
+
+    rb_require("strscan/strscan");
```

Comment:
We may need to use `rb_funcall(rb_mKernel, rb_intern("require"), 1, rb_str_new_cstr("strscan/strscan"))` or something because RubyGems may replace `rb_require()`.

Reply:
Are you sure? Other gems like `psych`, `json` etc use `rb_require` directly without known issues.

Reply:
Oh, I didn't know it.
OK. Let's use this. I hope that this is a temporary workaround...